task-libs/bosh/manifest_test.go

```diff
@@ -103,18 +103,18 @@ update:
     max_in_flight: 1
     update_watch_time: 1
 releases:
-  - name: release-a
-    sha1: ""
-    url: ""
-    version: ""
-  - name: release-b
-    sha1: ""
-    url: ""
-    version: ""
+    - name: release-a
+      sha1: ""
+      url: ""
+      version: ""
+    - name: release-b
+      sha1: ""
+      url: ""
+      version: ""
 stemcells:
-  - alias: default
-    os: some-os
-    version: 1.2.3
+    - alias: default
+      os: some-os
+      version: 1.2.3
```

Comment:
Is this valid YAML with this level of indentation?

Reply:
I've no idea if it's valid yaml, but it fixes the tests, which have been failing for some time as far as I can tell.